test/nuxeo-slots.html

```diff
@@ -195,6 +195,15 @@
     return container.querySelectorAll(':not(nuxeo-slot)');
   }
 
+  // asserts that all slot contributions are inserted before <nuxeo-slot>
+  function _assertPosition(elem) {
+    const parentChildren = Array.from(elem.children);
```

Comment:
nit: not sure it is the best name `parentChildren`. Maybe only `children` or `elementChildren`?

Reply:
Indeed, `parentChildren` is a silly name. I'd say `siblings` would perhaps be the most appropriate here.